### PR TITLE
Updating sales job ads

### PR DESCRIPTION
**Pull request by jesssummerfield**
@jackfranklin please can you check this?

fyi @benmiller169 @george-e-britton @moyni

### CODE REVIEW

File: app/pages/about/jobs/positions/account-executive.js [hunk 1]
```diff
@@ -62,11 +62,13 @@ export default class AccountExecutive extends React.Component {
...
-          someone with 1-3 years' professional experience in client-facing roles.
-          To find out more about what we look for when hiring, and how we train
-          new hires, visit <a href='https://gocardless.com/blog/how-we-train-salespeople/'>our
-          blog</a>.
+          Our current team come from a variety of backgrounds: straight out of
+          university, consulting and startups. To find out more about what we
+          look for when hiring, and how we train new hires, visit
+          <a href='https://gocardless.com/blog/how-we-train-salespeople/'> our
+          blog</a>, and don't miss our latest
+          <a href='https://gocardless.com/blog/the-account-executive-interview-process/'> post </a>
```
Comment (jackfranklin) [inline app/pages/about/jobs/positions/account-executive.js]: is it on purpose that you have the blank spaces inside the tag here?

**Comment by benmiller169** [comment]
@jesssummerfield loving your work - I'd do something like 'We've written a [blog post] which takes you through our interview process and gives you some tips. You can also read about the training you'll be getting in your first 4 weeks [here]. We look forward to hearing from you!'

**Comment by jackfranklin** [comment]
:+1:

**Comment by benmiller169** [comment]
woohoo, lvoe a good merge

-- 
Ben Miller
020 3637 2047

**Pull request merged**